tests/RestService.test.js

```diff
@@ -7,17 +7,23 @@ describe("RestService", () => {
   beforeAll(async () => {
     yaxysBuffer = global.yaxys
 
-    const adapterMethodsToEmulate = ["find", "findOne", "insert", "update", "count"]
+    const adapterMethodsToEmulate = ["find", "findOne", "insert", "update", "count", "delete"]
     global.yaxys = {
-      db: _.reduce(
-        adapterMethodsToEmulate,
-        (memo, methodName) => {
-          memo[methodName] = function() {
-            return [methodName, ...arguments]
-          }
-          return memo
+      db: Object.assign(_.reduce(
```

Comment:
Instead of Object.assign, you could just pass this additional object as a third argument into reduce